include/functions.h

```diff
@@ -1318,16 +1318,14 @@ GameStateFunc GameState_GetInit(GameState* gameState);
 u32 GameState_IsRunning(GameState* gameState);
 #if OOT_DEBUG
 void* GameState_Alloc(GameState* gameState, size_t size, char* file, s32 line);
-#endif
-void func_800C55D0(GameAlloc* this);
+
```

Comment:
nit: remove 3 extra newlines (seems the only place one is used here is between `boot` and `code`)